stacks-core/src/c32.rs

```diff
@@ -0,0 +1,393 @@
+use crate::crypto::hash::SHA256Hash;
+
+const C32_ALPHABET: &[u8; 32] = b"0123456789ABCDEFGHJKMNPQRSTVWXYZ";
+const C32_BYTE_MAP: [Option<u8>; 128] = [
```

Comment:
Could we represent this in a more compact/readable format? Or just create this from the alphabet?

Reply:
I've changed it so it's much more compact now, and `cargo fmt` won't mess with it. It can be generated but generation is not trivial. I believe there was a comment about generation somewhere. Also, I'm not sure if there are any performance implications when comparing const data and something lazily generated.

```rust
#[rustfmt::skip]
const C32_BYTE_MAP: [Option<u8>; 128] = [
    None,       None,       None,       None,       None,       None,       None,       None,
    None,       None,       None,       None,       None,       None,       None,       None,
    None,       None,       None,       None,       None,       None,       None,       None,
    None,       None,       None,       None,       None,       None,       None,       None,
    None,       None,       None,       None,       None,       None,       None,       None,
    None,       None,       None,       None,       None,       None,       None,       None,
    Some(0),    Some(1),    Some(2),    Some(3),    Some(4),    Some(5),    Some(6),    Some(7),
    Some(8),    Some(9),    None,       None,       None,       None,       None,       None,
    None,       Some(10),   Some(11),   Some(12),   Some(13),   Some(14),   Some(15),   Some(16),
    Some(17),   Some(1),    Some(18),   Some(19),   Some(1),    Some(20),   Some(21),   Some(0),
    Some(22),   Some(23),   Some(24),   Some(25),   Some(26),   None,       Some(27),   Some(28),
    Some(29),   Some(30),   Some(31),   None,       None,       None,       None,       None,
    None,       Some(10),   Some(11),   Some(12),   Some(13),   Some(14),   Some(15),   Some(16),
    Some(17),   Some(1),    Some(18),   Some(19),   Some(1),    Some(20),   Some(21),   Some(0),
    Some(22),   Some(23),   Some(24),   Some(25),   Some(26),   None,       Some(27),   Some(28),
    Some(29),   Some(30),   Some(31),   None,       None,       None,       None,       None,
];
```